### PR TITLE
Fixed failing tests in test_common_utils and test_algorithms

### PR DESCRIPTION
**Pull request by kajetanj**
## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Motivation and Context / Related issue

Two tests were failing:

1. `FAILED tests/core/test_common_utils.py::test_create_one_dim_tr_model - omegaconf.errors.ConfigAttributeError: Missing key target_is_delta`
2. `FAILED tests/algorithms/test_algorithms.py::test_mbpo - omegaconf.errors.ConfigAttributeError: Missing key save_video`

## How Has This Been Tested (if it applies)

`python -m pytest tests/core` and `python -m pytest tests/algorithms`

## Checklist

- [x] The documentation is up-to-date with the changes I made.
- [x] I have read the [**CONTRIBUTING**](../../CONTRIBUTING.md) document and completed the CLA (see **CONTRIBUTING**).
- [ ] All tests passed, and additional code has been covered with new tests.

I didn't run MuJoCo tests.

### CODE REVIEW

File: tests/core/test_common_utils.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def test_create_one_dim_tr_model():
         },
         "algorithm": {
             "learned_rewards": True,
-            "terget_is_delta": True,
+            "target_is_delta": True,
```
Comment (luisenp) [inline tests/core/test_common_utils.py:45]: Good catch! Was the test failing for you? This test runs fine for me even with the typo. Can you share the error you were getting? I wonder if the difference is due to a Hydra version (my version just sets `cfg.algorith.target_is_delta = None`.

Comment (luisenp) [reply tests/core/test_common_utils.py:45]: Oh sorry, I missed that you had posted the error in the description. Probably some changes in a new version of OmegaConf, thanks for the fix!

Comment (kajetanj) [reply tests/core/test_common_utils.py:45]: No problem, thanks for the library!

**Comment by luisenp** [comment]
@kajetanj Can you also manually add the changes in [this](https://github.com/facebookresearch/mbrl-lib/pull/90/files) PR, which fix CI tests for this one? (and that one doesn't pass due to the errors you are fixing here)

**Pull request merged**